### PR TITLE
[BI-63] created footer + a little restyling

### PR DESCRIPTION
**Pull request by LAlex14**
-  +author and posted info on blog page

image

### CODE REVIEW

File: src/components/PageFooter.vue [hunk 1]
```diff
@@ -0,0 +1,178 @@
+<template>
+  <div class="absolute bottotm-0 w-full">
+    <!-- <div class="h-16 bg-gradient-to-b from-white to-gray-100" /> -->
```
Comment (PascuMihai) [inline src/components/PageFooter.vue:3]: remove this

File: src/components/PageFooter.vue [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+          {{ $t('footer.rights.text') }}
+        </div>
+        <div class="flex space-x-1 text-gray-600">
+          <a
+            href="https://github.com/BinarCode/blog-intership"
+            target="_blank"
+            rel="noopener"
+            aria-label="Github Link"
+            class="p-1 sm:p-2 rounded-md  hover:text-indigo-500"
+            data-v-5e437550=""
```
Comment (PascuMihai) [inline src/components/PageFooter.vue:27]: remove this

File: src/components/PageFooter.vue [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+              stroke="currentColor"
+              stroke-width="2"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              class="feather feather-github"
+              data-v-5e437550=""
+            >
+              <path
+                d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
+                data-v-5e437550=""
```
Comment (PascuMihai) [inline src/components/PageFooter.vue:44]: remve unused code

File: src/views/Blog.vue [hunk 4]
```diff
@@ -54,15 +63,38 @@ export default {
   async mounted() {
     try {
       let data = await getBlog(this.blogId);
+      console.log(data);
```
Comment (PascuMihai) [inline src/views/Blog.vue:66]: please review the code before asking for review, remove this

File: src/views/Blog.vue [hunk 5]
```diff
@@ -54,15 +63,38 @@ export default {
...
     get,
+    getDate() {
+      let date = new Date(get(this.blog, 'attributes.created_at'));
+      const options = {
+        weekday: 'long',
+        year: 'numeric',
+        month: 'long',
+        day: 'numeric',
+      };
+      // let year = date.getFullYear();
```
Comment (PascuMihai) [inline src/views/Blog.vue:85]: remove this

**Review by Ceciliaa — APPROVED** [review]

**Review by PascuMihai — APPROVED** [review]

**Pull request merged**